src/mlpack/methods/softmax_regression/softmax_regression_impl.hpp

```diff
@@ -54,7 +55,7 @@ double SoftmaxRegression::Train(const arma::mat& data,
 
   // Train the model.
   Timer::Start("softmax_regression_optimization");
-  const double out = optimizer.Optimize(regressor, parameters);
+  const double out = optimizer.Optimize(regressor, parameters,callbacks...);
```

Comment:
```suggestion
  const double out = optimizer.Optimize(regressor, parameters, callbacks...);
```

Same here, please be sure to follow the style guide: https://github.com/mlpack/mlpack/wiki/DesignGuidelines#style-guidelines